src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -0,0 +1,180 @@
+src/nuget-client/test/NuGet.Core.Tests/NuGet.Packaging.Test/LicensesTests/LicenseExpressionTokenizerTests.cs
+src/nuget-client/test/NuGet.Core.Tests/NuGet.Packaging.Test/LicensesTests/NuGetLicenseExpressionParserTests.cs
+src/nuget-client/test/NuGet.Core.Tests/NuGet.Packaging.Test/LicensesTests/NuGetLicenseTests.cs
+src/nuget-client/test/TestUtilities/Test.Utility/JsonData.cs
+
+#
+# roslyn-analyzers
+#
+
+# Build asset, not applicable to source
+src/roslyn-analyzers/assets/EULA.txt|ms-net-library
+
+#
+# roslyn
+#
+
+# Test data
+src/roslyn/src/Analyzers/VisualBasic/Tests/FileHeaders/FileHeaderTests.vb|unknown-license-reference
+src/roslyn/src/EditorFeatures/CSharpTest2/EmbeddedLanguages/RegularExpressions/Regex_RealWorldPatterns.json
+
+# False positive
+src/roslyn/src/Compilers/**/Portable/Symbols/NonMissingModuleSymbol.*|proprietary-license
+
+#
+# runtime
+#
+
+# Doc describing licenses, not applicable to source
+src/runtime/docs/project/copyright.md
+src/runtime/docs/project/glossary.md
+
+# Doc that references a license, not applicable to source
+src/runtime/src/coreclr/nativeaot/docs/compiling.md|openssl-ssleay
+
+# Installer asset, not applicable to source
+src/runtime/src/installer/pkg/LICENSE-MSFT.TXT
+src/runtime/src/installer/pkg/THIRD-PARTY-NOTICES.TXT
+
+# False positive
+src/runtime/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicFileLicenseProvider.cs|proprietary-license
+src/runtime/src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs|other-permissive
+src/runtime/src/libraries/System.Net.Sockets/tests/FunctionalTests/UdpClientTest.cs|other-permissive
+src/runtime/src/libraries/System.Net.Sockets/tests/FunctionalTests/SendReceive/SendReceive.cs|other-permissive
+src/runtime/src/libraries/System.Private.Xml.Linq/tests/XDocument.Common/InputSpace.cs|other-permissive
+src/runtime/src/libraries/System.Private.Xml.Linq/tests/XDocument.Common/THIRD-PARTY-NOTICE|other-permissive
+src/runtime/src/libraries/System.Reflection.Metadata/tests/Resources/README.md|unknown-license-reference
+src/runtime/src/libraries/System.Runtime/tests/TestModule/README.md|unknown-license-reference
+src/runtime/src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/XmlLicenseTransform.cs|proprietary-license
+src/runtime/src/libraries/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs|unknown-license-reference
+src/runtime/src/mono/mono/mini/mini-posix.c|unknown-license-reference
+src/runtime/src/mono/mono/mini/mini-windows.c|unknown-license-reference
+src/runtime/src/native/external/libunwind/doc/libunwind-ia64.*|generic-exception
+src/runtime/src/tests/JIT/Performance/CodeQuality/V8/Crypto/Crypto.cs|unknown-license-reference
+
+# Test data
+src/runtime/src/libraries/System.ServiceModel.Syndication/tests/TestFeeds/AtomFeeds/*.xml
+src/runtime/src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/AttRegexTests.cs|other-permissive
```

Comment:
This one is considered as a variant of the MIT license: https://github.com/spdx/license-list-XML/pull/2058. We should probably submit this to scancode and to get that fixed up officially.

Reply:
Reported as https://github.com/nexB/scancode-toolkit/issues/3530